Makefile

```diff
@@ -287,6 +287,19 @@ golangci-lint:
 		echo "Skipping golangci-lint as not installed"; \
 	fi
 
+# golang doesn't always clean indirect dependencies in go.mod
+# we can remove all '// indirect' lines, and they will be re-added
+bump:
+	@echo "bump dependencies: "
+	@if [ "$$(uname)" = "Darwin" ]; then \
+		sed -i '' '/\/\/ indirect$$/d' go.mod; \
```

Comment:
Probably best to simplify this using ddev/ddev-utilities so it's the same command?

Or maybe awk would be easier.